python/ray/rllib/train.py

```diff
@@ -35,17 +35,7 @@
     args = parser.parse_args()
     json_config = json.loads(args.config)
 
-    if args.redis_address:
-        # TODO(pcm): This is a hack, provide a better solution
-        if args.alg == "A3C":
-            num_workers = a3c.DEFAULT_CONFIG["num_workers"]
-            num_workers = json_config.get("num_workers", num_workers)
-            ray.init(redis_address=args.redis_address,
-                     num_cpus=num_workers)
```

Comment:
yeah, if there are only 3 CPUs, then we won't run more than 3 tasks concurrently (this includes both remote functions and actor methods)